clap_blocks/src/catalog_dsn.rs

```diff
@@ -36,17 +62,44 @@ pub struct CatalogDsnConfig {
     #[clap(
         long = "--catalog-max-connections",
         env = "INFLUXDB_IOX_CATALOG_MAX_CONNECTIONS",
-        default_value = "10"
+        default_value = default_max_connections(),
     )]
     pub max_catalog_connections: u32,
 
     /// Schema name for PostgreSQL-based catalogs.
     #[clap(
         long = "--catalog-postgres-schema-name",
         env = "INFLUXDB_IOX_CATALOG_POSTGRES_SCHEMA_NAME",
-        default_value = iox_catalog::postgres::SCHEMA_NAME,
+        default_value = PostgresConnectionOptions::DEFAULT_SCHEMA_NAME,
     )]
     pub postgres_schema_name: String,
+
+    /// Set the amount of time to attempt connecting to the database.
+    #[clap(
+        long = "--catalog-connection-timeout",
+        env = "INFLUXDB_IOX_CATALOG_CONNECT_TIMEOUT",
+        default_value = default_connection_timeout(),
+        parse(try_from_str = humantime::parse_duration),
+    )]
+    pub connect_timeout: Duration,
```

Comment:
```suggestion
    pub connection_timeout: Duration,
```

Maybe to make the struct field more consistent with the parameter name?

Reply:
Oh, good catch. It should be `connect` timeout for both since it esp. concerns the "connect" action, not the entire connection lifecycle.